CMIP6/Guide/dataManagers.md

```diff
@@ -100,6 +100,8 @@ This section describes the usage of the esgprep tool. This tool will take of you
 
 This section describe steps required to publish the first version of a new CMIP6 dataset. After those steps your CMIP6 dataset will be findable and accessible from the ESGF portal.
 
+Note on publishing: please do not manually edit esg.cmip6.ini to add missing information if your data files won't publish.  It is likely that either the model is unregistrered or there is an error in the metadata that needs to be corrected.  Please (1) contact the modeling group to ensure all fields are properly registered (2) support from CDNOT to get the publisher config (esg.cmip6.ini file) updated.   
```

Comment:
`registrered` -> unregistered.

It might also be worthwhile calling out the CMIP6_CVs repo specifically, so that it's crystal clear that all model data that are being published must be associated with a known registered entry (institution_id, source_id etc)